Packages/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "dependencies": {
+    "com.unity.ide.rider": "1.1.4",
+    "com.unity.ide.vscode": "1.2.1",
+    "com.unity.textmeshpro": "2.1.1",
+    "com.unity.ugui": "1.0.0",
+    "com.unity.modules.ai": "1.0.0",
+    "com.unity.modules.androidjni": "1.0.0",
+    "com.unity.modules.animation": "1.0.0",
+    "com.unity.modules.assetbundle": "1.0.0",
+    "com.unity.modules.audio": "1.0.0",
```

Comment:
as I understood these dependencies will be added with the package, won't they?
may it be better to exclude most of them since they look unused?

Reply:
No, these dependencies are only for the development unity project and these dependencies are default for any newly created unity project.

Package dependencies(if any) are listed in `Packages/DependenciesHunter/package.json`